README.md

```diff
@@ -33,6 +33,27 @@ You can simply install ROMA and dependency libralies by using a "gems" command o
 $ gem install roma
 ```
 
+### Troubleshooting
```

Comment:
I think this part should be a part of `Installation` and found it from [another document](http://roma-kvs.org/learn/tokyocabinet_lib.html). I tried to install `roma` from this repository like you into Ubuntu 14.04(trusty). `lbibz2-dev` is not installed in default. so installing `libbz2-dev` is required before installing `roma`. so it's not a trouble just a prerequisite for installing `roma` into Ubuntu. 

I will merge your PR with the branch 'master'  after update it. after that, I will write installation for tokyocabinet-ruby since it looks hard for visiters to this repository to be aware of another document site...